ibm-eventstore/src/main/scala/akka/stream/alpakka/ibm/eventstore/scaladsl/EventStore.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.ibm.eventstore.scaladsl
+
+import scala.concurrent.Future
+
+import akka.Done
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+import akka.stream.scaladsl.Keep
+import akka.stream.scaladsl.Sink
+
+import com.ibm.event.oltp.EventContext
+import com.ibm.event.oltp.InsertResult
+import org.apache.spark.sql.Row
+
+object EventStoreSink {
+
+  /**
+   * Creates a sink for insertion of records into EventStore.
+   * @param databaseName Name of the database, database has to exist before the call to this function.
+   * @param tableName    Name of the table, database has to exist before the call to this function.
```

Comment:
What happens if database/table does not, is the behavior undefined or does the stream fail upon materialization?

Reply:
No, it throws before materialization when you try to lookup the table with ` EventContext.getEventContext(databaseName)/ context.getTable(tableName)

Reply:
Ok, that is a likely a bit surprising, please add that to the scaladoc. 

I think you can skip the `@param`s as those are fairly self-explanatory and just have a paragraph saying that this will throw if the database or table does not exist when calling this factory method.